app/assets/stylesheets/components/_personal-key.scss

```diff
@@ -2,7 +2,7 @@
   position: relative;
 
   &::before {
```

Comment:
Is this a duplicate style from the other in `_modal.scss`. I see both are being applied to the element, it seems like we should only have one.

![image](https://user-images.githubusercontent.com/1779930/191511912-4085fa52-9f0f-44b7-9c06-57b643a17c96.png)